README.md

```diff
@@ -152,7 +152,7 @@ import (
 var ctx = context.Background()
 
 func ExampleClient() {
-    url := "redis://localhost:6379?password=hello&protocol=3"
+    url := "redis://user:secret@localhost:6379/0?protocol=3"
```

Comment:
Maybe change `secret` to "password" or something more indicative?
WDYT?

Reply:
You are right, I will change it ASAP.

Reply:
> Maybe change `secret` to "password" or something more indicative? WDYT?

DONE @ofekshenawa

Reply:
Thank you!